text/0000-std-rpc-api.md

```diff
@@ -198,6 +201,7 @@ void XyzwServerRpc(int framekey, ServerRpcParams rpcParams = default) { /* ... *
 ```cs
 struct ClientRpcSendParams
 {
+    // who are the target clients?
     ulong[] TargetClientIds;
```

Comment:
Using an `IEnumerable<ulong>` here would be very convenient for developers, that way i can use a List, Array, NativeArray or even the results of a LINQ query. Accepting an IEnumerable allow us to pass in any collection, as long as it contains identifiers. Additional care has to be taken by the framework not to iterate the targets too often, in the case of a LINQ query lazy evaluation can make this costly.